scrunch/datasets.py

```diff
@@ -1115,7 +1152,7 @@ def combine_multiple_response(self, variable, map, categories=None, default=None
         responses into new categorized ones
 
             map={
-                1: 1,
+                1: [1],
```

Comment:
why this? From what I understand, doug and users don't like to use lists where it isn't necessary.

if you ask me, I'd love to require single element lists for consistency sake, but I recall doug's example's never liking to do single element lists.